docs/blog/2024-07-17-release-1.0.0.md

```diff
@@ -0,0 +1,27 @@
+---
+slug: release-1.0.0
+title: Release 1.0.0
+authors: [netdpb]
+tags: []
+---
+
+We are pleased to announce that
+[JSpecify 1.0](https://repo.maven.apache.org/maven2/org/jspecify/jspecify/1.0.0/)
```

Comment:
```suggestion
[JSpecify 1.0.0](https://repo.maven.apache.org/maven2/org/jspecify/jspecify/1.0.0/)
```

...just to be consistent and to make clearer to users what they will need to paste into build scripts.